src/client/components/UserList/UserList.tsx

```diff
@@ -116,15 +119,46 @@ const UserRow: React.FC<{ user: User; showEmail: boolean }> = ({ user, showEmail
   )
 }
 
-const UsersTableHeadRow: React.FC<{ showEmail: boolean }> = ({ showEmail }) => {
-  const { i18n } = useTranslation()
+const AdminTableUserRow: React.FC<{ user: User }> = ({ user }) => {
+  const { t } = useTranslation()
+
+  const { id, status } = user
+
+  return (
+    <tr
+      className={classNames({
+        'user-list__inactive-user': status === UserStatus.inactive,
+      })}
+    >
+      <UserColumn user={user} field="name" />
+      <td className="user-list__cell">
+        <div className="user-list__cell--read-only">TODO</div>
+      </td>
+      <UserColumn user={user} field="email" />
+      <td className="user-list__cell user-list__edit-column">
+        <Link to={ClientRoutes.Admin.User.getLink({ id })} type="button" className="link">
+          {t('userManagement.edit')}
+        </Link>
+      </td>
+    </tr>
+  )
+}
+
+const UsersTableHeadRow: React.FC<{ showLoginEmail: boolean; showRole: boolean; showRoles: boolean }> = ({
+  showLoginEmail,
+  showRole,
+  showRoles,
+}) => {
+  const { t } = useTranslation()
```

Comment:
Split each component in single files

Reply:
<img width="374" alt="Schermata 2022-11-02 alle 15 28 28" src="https://user-images.githubusercontent.com/10047945/199515954-00f9d4f2-1a9c-49ae-847e-85556529a7c9.png">